examples/web/index.html

```diff
@@ -501,6 +501,339 @@ <h2 class="typography-heading-medium-text">Alert</h2>
+                width="65"
+                height="24"
+                alt="Spirit"
+              />
+            </a>
+            <div class="Header__toggle">
+              <button
+                type="button"
+                class="Button Button--inverted"
+                data-toggle="header"
+                data-target="#header-example-1__bar"
+                aria-controls="header-example-1__bar"
+                aria-expanded="false"
+              >
+                <span class="Header__icon Header__icon--menu" aria-hidden="true"></span>
+                <span class="accessibility-hidden">Menu</span>
+              </button>
+            </div>
+            <nav id="header-example-1__bar" class="Header__bar">
+              <div class="Header__content">
+                <button
+                  type="button"
+                  class="Header__close"
+                  data-toggle="header"
+                  data-target="#header-example-1__bar"
+                  aria-controls="header-example-1__bar"
+                  aria-expanded="false"
+                >
+                  <span class="Header__icon Header__icon--close" aria-hidden="true"></span>
+                  <span class="accessibility-hidden">Close</span>
```

Comment:
I know it is just an example, but we mix here Czech and English. Shouldn't we use only one language in one component?